pkgs/development/python-modules/torchmetrics/default.nix

```diff
@@ -22,10 +24,12 @@ let
 in
 buildPythonPackage {
   inherit pname version;
+  pyproject = true;
 
+  # The repo was moved from PyTorchLightning/metrics to Lightning-AI/torchmetrics
```

Comment:
Do we need to persist that information here?

Reply:
It's an explanation on that the repo was moved and why the owner and repo are different.
I don't think this needs to be changed.

Reply:
This is really more helpful as a transient argument in the commit message. I'd expect the commit message to answer the "Why?" bits.

Reply:
Done.